src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -102,8 +102,9 @@ private URI checkDockerHostScheme(URI dockerHost) {
     private String checkDockerCertPath(boolean dockerTlsVerify, String dockerCertPath) {
         if (dockerTlsVerify) {
             if (StringUtils.isEmpty(dockerCertPath)) {
-                throw new DockerClientException(
-                        "Enabled TLS verification (DOCKER_TLS_VERIFY=1) but certifate path (DOCKER_CERT_PATH) is not defined.");
+                return dockerCertPath;
+//                throw new DockerClientException(
```

Comment:
I think anybody would be glad to fill configuration only in single DockerClientBuilder and get connection :)


> As I said: I really don't like this.

Didn't get what exactly. Making one builder, adding boolean for now while there is no better solution to unblock programmatic usage or having multiple configs/builders?